test/cloudnative/network_problems.go

```diff
@@ -43,7 +44,7 @@ func NetworkProblems(t *testing.T) features.Feature {
 	createNetworkProblems.Setup(manifests.InstallFromFile(csiNetworkPolicy))
 	createNetworkProblems.Setup(manifests.InstallFromFile(sampleNSPath))
 	createNetworkProblems.Setup(secrets.ApplyDefault(secretConfigs[0]))
-	createNetworkProblems.Setup(operator.InstallAllForKubernetes())
+	createNetworkProblems.Setup(operator.InstallAll(os.Getenv(operator.PlatformEnv)))
```

Comment:
choosing the platform based on the env var could be refactored into the test/operator package since as far as I can see it is only used for the operator anyway